app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
I just pushed 30d1f19 and 29c03ab, but I am getting a webmock error in `spec/models/employee_spec.rb`. I'm not sure why webmock isn't catching it. My only guess is it has something to do with this not being a `feature` spec, but I'm not exactly sure what the best practice around this would be. Thoughts?